src/Marker.tsx

```diff
@@ -36,9 +40,10 @@ const Marker: React.FC = () => {
   }, [])
   return (
     <Layer type="symbol" id="marker" layout={{ 'icon-image': 'marker-15' }}>
-      {coordinates.map((row) => {
-        return <Feature key="row" coordinates={[row.long, row.lat]} />
-      })}
+      {coordinates &&
+        coordinates.map((row) => {
+          return <Feature key="row" coordinates={[row.long, row.lat]} />
+        })}
```

Comment:
Misalignment of braces. Prettier would have caught this.

Reply:
❌ - Still need to fix this. I checked that the .prettierrc file is unchanged from the main branch, and formatted the marker.tsx file, no changes.  Might it be because I added the 'coordinates &&'?